spoon-runner/src/test/java/com/squareup/spoon/SpoonCoverageMergerTest.kt

```diff
@@ -11,7 +11,8 @@ import java.lang.String.format
 class SpoonCoverageMergerTest {
   @JvmField @Rule val testFolder = TemporaryFolder()
 
-  @Test fun shouldMergeCoverageFiles() {
+  @Test
+  fun shouldMergeCoverageFiles() {
```

Comment:
hmmmmmmmMMMMMMMmmm

Reply:
i realized this was probably updated in SquareAndroid codestyle, so I just updated and this no longer happens, which is good.

But the new code style apparently calls for:
```
  private fun createTemporaryCoverageFiles(serialId1: String, serialId2: String) {
```
to be 
```
  private fun createTemporaryCoverageFiles(
    serialId1: String,
    serialId2: String
  ) {
```
wtf

Reply:
It should only do that when the first thing doesn't fit on one line.

Reply:
Opened https://github.com/square/java-code-styles/issues/58 but I'm not sure who to tag, I'm not an admin on that project